node/pkg/db/governor_test.go

```diff
@@ -439,38 +466,75 @@ func TestLoadingOldPendingTransfers(t *testing.T) {
 	defer db.Close()
 	defer os.Remove(dbPath)
 
-	tokenBridgeAddr, err := vaa.StringToAddress("0x0290fb167208af455bb137780163b7b7a9a10c16")
+	ethereumTokenBridgeAddr, err := vaa.StringToAddress("0x0290fb167208af455bb137780163b7b7a9a10c16")
+	require.NoError(t, err)
+
+	bscTokenBridgeAddr, _ := vaa.StringToAddress("0x26b4afb60d6c903165150c6f0aa14f8016be4aec")
 	require.NoError(t, err)
 
 	tokenAddr, err := vaa.StringToAddress("0x707f9118e33a9b8998bea41dd0d46f38bb963fc8")
 	require.NoError(t, err)
 
-	xfer1 := &Transfer{
+	oldXfer1 := &Transfer{
 		Timestamp:      time.Unix(int64(1654516425), 0),
 		Value:          125000,
 		OriginChain:    vaa.ChainIDEthereum,
 		OriginAddress:  tokenAddr,
 		EmitterChain:   vaa.ChainIDEthereum,
-		EmitterAddress: tokenBridgeAddr,
-		MsgID:          "2/0000000000000000000000000290fb167208af455bb137780163b7b7a9a10c16/789101112131415",
-		Hash:           "Hash1",
+		EmitterAddress: ethereumTokenBridgeAddr,
+		// Don't set TargetChain or TargetAddress.
+		MsgID: "2/0000000000000000000000000290fb167208af455bb137780163b7b7a9a10c16/789101112131415",
+		Hash:  "Hash1",
 	}
 
-	err = db.StoreTransfer(xfer1)
+	err = db.storeOldTransfer(oldXfer1)
 	require.Nil(t, err)
 
-	xfer2 := &Transfer{
-		Timestamp:      time.Unix(int64(1654516430), 0),
+	newXfer1 := &Transfer{
+		Timestamp:      time.Unix(int64(1654516426), 0),
 		Value:          125000,
 		OriginChain:    vaa.ChainIDEthereum,
 		OriginAddress:  tokenAddr,
 		EmitterChain:   vaa.ChainIDEthereum,
-		EmitterAddress: tokenBridgeAddr,
+		EmitterAddress: ethereumTokenBridgeAddr,
+		TargetChain:    vaa.ChainIDBSC,
+		TargetAddress:  bscTokenBridgeAddr,
 		MsgID:          "2/0000000000000000000000000290fb167208af455bb137780163b7b7a9a10c16/789101112131416",
+		Hash:           "Hash1",
+	}
+
+	err = db.StoreTransfer(newXfer1)
+	require.Nil(t, err)
```

Comment:
nit: using `NoError()` instead of `Nil()` here is probably better for the explicitness, though they are functionally the same.